test/evergreen.yml

```diff
@@ -232,7 +225,7 @@ functions:
         echo "startup --output_user_root=${workdir}/bazel-output-root" >> ~/.bazelrc
         echo "BAZELISK_HOME=${workdir}/bazelisk_home" >> ~/.bazeliskrc
 
-        bazel build --mongo_toolchain_version=v4 --config=opt --build_enterprise=False install-mongod
+        bazel build --mongo_toolchain_version=v5 --config=opt --build_enterprise=False install-mongod
```

Comment:
Mongo's build system now mandates the v5 toolchain. Running with the original v4 results in the following error:
`@@//bazel/config:mongo_toolchain_version mongo_toolchain_version allowed to take values {v5} but was set to unallowed value v4`